src/Uno.Extensions.Authentication.Oidc/build/Package.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <ItemGroup Condition=" '$(ImplicitUsings)' == 'true' OR '$(ImplicitUsings)' == 'enable' ">
+    <Using Include="Uno.Extensions" />
```

Comment:
I have a feeling this may cause collisions with multiple Using Includes for the same namespace. We should only be adding namespaces unique to that library. Effectively everything in Extensions has a dependency on Core which can add the using for Uno.Extensions.

Reply:
I can adjust this but tbh if implicit usings are that fragile it's going to be a royal pita to manage which libraries import which usings.